applications/IgaApplication/tests/test_IgaApplication.py

```diff
@@ -38,5 +38,5 @@ def AssembleTestSuites():
     return suites
 
 if __name__ == '__main__':
-    #run_cpp_unit_tests.run()
+    run_cpp_unit_tests.run()
```

Comment:
```suggestion
```

this should not be here, I am working on a better solution

=> note that the CI anyway runs the cpp tests in a different way so having this here runs them twice (I know this is in many apps but its legacy)

Reply:
ok. I am fine with removing it here then.